packages/k2-alpine/src/hooks/usePressableGesture.ts

```diff
@@ -10,20 +10,23 @@ import {
   withSpring,
   withTiming
 } from 'react-native-reanimated'
+import { AnimatedPressableProps } from '../components'
 import { ANIMATED } from '../utils'
 
 const SCROLL_THRESHOLD = 1 // pixels
 
 /**
  * Use this hook to handle pressable gestures.
  * @param callback - The callback to call when the gesture is completed.
+ * @param props - The props to pass to the component.
  * @returns An object with the animatedStyle, onTouchStart, onTouchMove, onTouchCancel, and onTouchEnd properties.
  *
  * **NOTE** can be used on any component that supports the GestureResponderEvent and GestureTouchEvent props.
  * **EXAMPLE** GestureDetector, Pressable, TouchableOpacity, etc.
  */
 export function usePressableGesture(
-  callback?: (event: GestureResponderEvent) => void
+  callback?: (event: GestureResponderEvent) => void,
+  props?: AnimatedPressableProps
```

Comment:
do you think we actually need the whole AnimatedPressableProps here or just some? looks like only `disabled` is being looked at